v2/objective-c/.gitignore

```diff
@@ -8,10 +12,17 @@ build/
 *.perspectivev3
 !default.perspectivev3
 xcuserdata
+
+# Other
 *.xccheckout
 *.moved-aside
-DerivedData
+*.xcuserstate
+*.xcscmblueprint
+
+# Obj-C/Swift specific
 *.hmap
 *.ipa
-*.xcuserstate
 
+# CocoaPods
```

Comment:
This looks great.

You've got a TODO around the readme --- if you add that, or decide you don't want it, I'd be happy to give a ship-it.